yellowbrick/classifier/prcurve.py

```diff
@@ -377,11 +382,17 @@ def precision_recall_curve(model, X, y, ax=None, train_size=0.8,
 
     X : ndarray or DataFrame of shape n x m
         A feature array of n instances with m features the model is trained on.
-        This array will be split into train and test splits.
+        This array will be split into train and test splits if X_test is not specified.
 
     y : ndarray or Series of length n
         An array or series of target or class values. This vector will be split
-        into train and test splits.
+        into train and test splits  if y_test is not specified.
+
+    X_test : ndarray or DataFrame of shape n x m
+        A feature array of n instances with m features that the model is tested on.
+        
+    y_test : ndarray or Series of length n
+        An array or series of target or class values that serve as actual labels for X_test.
```

Comment:
Same as above, would you mind adding some language about how this is optional?

Reply:
done